src/Domain/UseCase/AddResourceVote/Command.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Domain\UseCase\AddResourceVote;
+
+class Command
+{
+    public $slug;
+    public $resource_id;
```

Comment:
use `camelCase` instead of `snake_case` according to PSR-1 http://www.php-fig.org/psr/psr-1/